services/clusters-keeper/tests/unit/test_utils_clusters.py

```diff
@@ -175,13 +185,81 @@ def test_create_deploy_cluster_stack_script(
         for i in dict_settings
     )
 
+    # check that the RabbitMQ settings are null since rabbit is disabled
+    assert re.search(r"AUTOSCALING_RABBITMQ=null", deploy_script)
+
     # check the additional tags are in
     assert all(
         f'"{key}": "{value}"' in deploy_script
         for key, value in additional_custom_tags.items()
     )
 
 
+@pytest.fixture(
+    params=["default", "custom"], ids=["defaultRabbitMQ", "specialClusterRabbitMQ"]
+)
+def rabbitmq_settings_fixture(
+    app_environment: EnvVarsDict,
+    enabled_rabbitmq: RabbitSettings,
+    request: pytest.FixtureRequest,
+    monkeypatch: pytest.MonkeyPatch,
+    faker: Faker,
+) -> RabbitSettings | None:
+    if request.param == "custom":
+        # Create random RabbitMQ settings using faker
+        custom_rabbit_settings = RabbitSettings(
```

Comment:
TIP: use `RabbitSettings.model_json_schema()["examples"]` as fixtures

Reply:
we do not have examples there. I can add one though